src/datasource.js

```diff
@@ -107,11 +109,19 @@ export default class Datasource {
     }
   }
 
+  transformDate = value => {
+    let d = Date.parse(value)
+    // handle timestamp as duration
+    if (d < this.maxDurationYear) {
+      return this.maxDurationYear - d
```

Comment:
That's incorrect, it should be `return d - this.epochYear` (where epoch is 1970-01-01).

Reply:
And probably you should verify that `d` is not before epoch.

Reply:
fixed.